### PR TITLE
Add test for ethstore-cli

### PR DESCRIPTION
**Pull request by debris**
replaces #8081, where the CI cache is broken

### CODE REVIEW

**Comment by izgzhen** [comment]
the CI failed again ... I don't know why it might happen, but maybe you can try rebase it again.

**Comment by debris** [comment]
@paritytech/ci can you help here? I restarted the build / added more commits / rebased pr and it still fails with the same unrelated error...

File: ethstore/cli/tests/cli.rs [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+#[test]
+fn cli_cmd() {
+	Command::new("cargo")
+		.arg("build")
+		.output()
+		.unwrap();
+
+	let dir = TempDir::new("test-vault").unwrap();
+	println!("tempdir: {}", dir.path().to_str().unwrap());
```
Comment (rphmeier) [inline ethstore/cli/tests/cli.rs]: stray println

Comment (izgzhen) [reply ethstore/cli/tests/cli.rs]: @debris looks like I can't push to this branch, can you help rework the patch here?

Comment (debris) [reply ethstore/cli/tests/cli.rs]: sure!  fixed, rebased and put in a new branch again!

**Comment by 5chdn** [comment]
This was merged in #8027 right?

**Comment by debris** [comment]
@5chdn no, it wasn't :/

**Comment by debris** [comment]
we should reopen it once pr is fixed an CI issues no longer exist

**Comment by izgzhen** [comment]
this commit? https://github.com/paritytech/parity/commit/6e49ff1d98cdcc52733f9ea93dd95a6d487b3279

File: test.sh [hunk 2]
```diff
@@ -3,31 +3,43 @@
...
 ./scripts/validate_chainspecs.sh
+fi
 
-cargo test -j 8 $OPTIONS --features "$FEATURES" --all --exclude evmjit $1
+# Running test's
+echo "________Running Parity Full Test Suite________"
 
-# Validate --no-default-features build
-cargo check --no-default-features
+cargo test -j 8 $OPTIONS --features "$FEATURES" --all --exclude evmjit $1
```
Comment (niklasad1) [inline test.sh]: You can remove `evmjit` here it is removed

**Comment by debris** [comment]
already merged

**Pull request closed without merging**